Spark/metrics.csv

```diff
@@ -0,0 +1,207 @@
+spark.stage.executor_run_time.max,gauge,,millisecond,,The max executor run time in the application's stages,0,spark,stg exe rn tm mx
+spark.stage.executor_run_time.avg,gauge,,millisecond,,The average executor run time in the application's stages,0,spark,stg exe rn tm avg
+spark.stage.executor_run_time.median,gauge,,millisecond,,The median executor run time in the application's stages,0,spark,stg exe rn tm med
+spark.stage.executor_run_time.95percentile,gauge,,millisecond,,The 95th percentile executor run time in the application's stages,0,spark,stg exe rn tm 95
+spark.stage.executor_run_time.count,rate,,millisecond,,The number of times the executor run time in the application's stages was sampled,0,spark,stg exe rn tm cnt
+spark.stage.input_bytes.max,gauge,,byte,,The max input bytes in the application's stages,0,spark,stg in byt mx
+spark.stage.input_bytes.avg,gauge,,byte,,The average input bytes in the application's stages,0,spark,stg in byt avg
+spark.stage.input_bytes.median,gauge,,byte,,The median input bytes in the application's stages,0,spark,stg in byt med
+spark.stage.input_bytes.95percentile,gauge,,byte,,The 95th percentile input bytes in the application's stages,0,spark,stg in byt 95
+spark.stage.input_bytes.count,rate,,byte,,The number of times the input bytes in the application's stages was sampled,0,spark,stg in byt cnt
+spark.stage.input_records.max,gauge,,record,,The max input records in the application's stages,0,spark,stg in rec mx
+spark.stage.input_records.avg,gauge,,record,,The average input records in the application's stages,0,spark,stg in rec avg
+spark.stage.input_records.median,gauge,,record,,The median input records in the application's stages,0,spark,stg in rec amed
+spark.stage.input_records.95percentile,gauge,,record,,The 95th percentile input records in the application's stages,0,spark,stg in rec 95
+spark.stage.input_records.count,rate,,record,,The number of times the input records in the application's stages was sampled,0,spark,stg in rec cnt
+spark.stage.output_bytes.max,gauge,,byte,,The max output bytes in the application's stages,0,spark,stg out byt mx
+spark.stage.output_bytes.avg,gauge,,byte,,The average output bytes in the application's stages,0,spark,stg out byt avg
+spark.stage.output_bytes.median,gauge,,byte,,The median output bytes in the application's stages,0,spark,stg out byt med
+spark.stage.output_bytes.95percentile,gauge,,byte,,The 95th percentile output bytes in the application's stages,0,spark,stg out byt 95
+spark.stage.output_bytes.count,rate,,byte,,The number of times the output bytes in the application's stages was sampled,0,spark,stg out byt cnt
+spark.stage.output_records.max,gauge,,record,,The max output records in the application's stages,0,spark,stg out rec mx
+spark.stage.output_records.avg,gauge,,record,,The average output records in the application's stages,0,spark,stg out rec avg

```

Comment:
let's correct "The average output records" to "The average number of output records". The same for other metrics with input/output records.